<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSwiftCore</class>
 <widget class="BlackGui::CSystemTrayWindow" name="CSwiftCore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>swiftcore</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="2">
     <widget class="QGroupBox" name="gb_Controls">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pb_startCore">
         <property name="styleSheet">
          <string notr="true">/** Main window **/
QTextEdit {
  background-color: DimGray;
}</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pb_stopCore">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">/** Main window **/
QTextEdit {
  background-color: DimGray;
}</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="BlackGui::Components::CInfoBarStatusComponent" name="comp_InfoBarStatus" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="BlackGui::Components::CLogComponent" name="comp_log" native="true"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="gb_dbusMode">
      <property name="title">
       <string>DBus Mode</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rb_sessionBus">
         <property name="text">
          <string>Session</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="rb_p2pBus">
         <property name="text">
          <string>P2P</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="rb_systemBus">
         <property name="text">
          <string>System</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="le_p2pAddress">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>P2P address</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="BlackGui::CCommandInput" name="le_CommandLineInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CSystemTrayWindow</class>
   <extends>QMainWindow</extends>
   <header>blackgui/systemtraywindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CInfoBarStatusComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/infobarstatuscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CLogComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/logcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CCommandInput</class>
   <extends>QLineEdit</extends>
   <header>blackgui/commandinput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
